docs/user/corrections/inelastic_incoherent.rst

```diff
@@ -0,0 +1,272 @@
+.. _user.corrections.inelastic_incoherent:
+
+Inelastic incoherent correction
+===============================
+
+Background
+----------
+
+In SANS, the focus is on the structural information obtained from elastic scattering, and
+inelastic scattering contributions are typically assumed to be negligible, however, this assumption
+does not always hold. For example, in samples containing hydrogen, which has a high
+incoherent scattering cross section, there can be a significant and wavelength-dependent
+contribution from inelastic scattering [Do2013]_.
+
+Due to the wavelength dependence of inelastic scattering effects, correcting for them is of
+particular concern for time-of-flight (TOF) pulsed neutron instruments like EQ-SANS, which use a
+broad spectrum of neutron wavelengths. The data reduction for EQ-SANS, therefore, includes optional
+wavelength-dependent corrections for inelastic incoherent scattering.
+
+Elastic reference normalization
+-------------------------------
+
+Before data reduction, the various data sets must be normalized to equivalent beam exposure.
+The normalization method typically used at EQ-SANS is proton charge on target and measured flux
+spectrum, which requires a pre-measured flux spectrum :math:`\phi(\lambda)`.
+However, the wavelength distribution of scattered neutrons can be blurred due to
+wavelength-dependent coherent-inelastic processes, as illustrated in the diagram below.
+Elastic reference normalization introduces a wavelength-dependent scale factor, `K`,
+to compensate for differences in the intensity scale resulting from these effects.
+
+.. figure:: /user/media/blurring_wavelength_distribution.jpg
+   :alt: blurring wavelength distribution
+   :width: 800px
+
+Procedure
+.........
+
+The following steps describe the elastic reference normalization available in `drtsans`:
+
+#. Get :math:`I(q,\lambda_i)` of the given elastic scattering reference run, making sure q-bins are
```

Comment:
Can you get from the CIS or IS the reason why this reference run is termed "elastic"? If this run was also taken at the  beamline, won't it suffer from the same blurring artifacts in the same way as the other runs do? What makes this run "special" or artifact free?